src/partials/EventDetail.js

```diff
@@ -82,19 +82,11 @@ export default function EventDetail({ open, setOpen, event }) {
           </div>
           <div className="mb-3 flex flex-col">
             <h3 className="font-medium mb-2 text-gray-500" href="#0">{`${event?.participant_ids.length}/${event?.max_participant_count} joined`}</h3>
-            <ul className="space-y-2">
-              {
-                event?.participant_ids.map(id => accounts.entities[id]).map(({ id, real_name, username }) => (
-                  <li key={id}>
-                    <AvatarAndName real_name={real_name} username={username} />
-                  </li>
-                ))
-              }
-            </ul>
+            {event?.participant_ids && <AvatarExpandable participants={event?.participant_ids.map(id => accounts.entities[id])} />}
```

Comment:
```suggestion
            <AvatarExpandable participants={event?.participant_ids.map(id => accounts.entities[id]) ?? []} />
```
This also works and (I guess) it's more stable.